### PR TITLE
Announcement: Secondary DNS IP address changes

### PR DESCRIPTION
**Pull request by AGS4NO**
Announcement for secondary DNS IP address changes.

### CODE REVIEW

File: content/articles/announcement-secondary-dns-ip-address-changes.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+* All zone transfers must be requested from the new IP addresses.
+* Any domain in your account with secondary DNS enabled will stop receiving updates unless the new IP addresses are added to the secondary DNS provider or server configuration.
+
+## What do you need to do?
+
+Review any domains you own with secondary DNS enabled, where DNSimple acts as the primary server used for outbound notifications and zone transfers.
+
+Update your secondary DNS provider or server configuration:
+
+1. Allow NOTIFY messages from the new IP addresses:
```
Comment (weppos) [inline content/articles/announcement-secondary-dns-ip-address-changes.md:43]: It is unclear that this is _in addition_ to the existing one. It becomes kind of implicit as you read number (3) but we should make it a bit more explicit.

**Review by itsalyse — APPROVED** [review]

File: content/articles/announcement-secondary-dns-ip-address-changes.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+</info>
+
+### Table of Contents {#toc}
+
+* TOC
+{:toc}
+
+---
+
+As part of our commitment to providing streamlined and reliable DNS management services, we are modernizing our secondary DNS infrastructure and updating the primary server IP addresses used for outbound notifications and zone transfers. **The current IPv4 address (50.31.225.92) will be retired beginning [DATE], 2024.**
```
Comment (weppos) [inline content/articles/announcement-secondary-dns-ip-address-changes.md:20]: Remember to set the DATE.

File: content/articles/announcement-secondary-dns-ip-address-changes.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
+---
+title: Secondary DNS IP Address Changes
+excerpt: DNSimple will be updating the IP addresses used for Secondary DNS outbound notifications and zone transfers. The current IPv4 address (50.31.225.92) will be retired beginning [DATE], 2024. Affected customers will be notified via email.
```
Comment (weppos) [inline content/articles/announcement-secondary-dns-ip-address-changes.md:3]: Remember to set the DATE.

Comment (san983) [reply content/articles/announcement-secondary-dns-ip-address-changes.md]: Set in https://github.com/dnsimple/dnsimple-support/pull/1258/commits/9f0f9f294517887d36cfa99483ccf0c87282f998

File: content/articles/announcement-secondary-dns-ip-address-changes.md (hunk 2 above)
Comment (san983) [reply content/articles/announcement-secondary-dns-ip-address-changes.md]: Set in https://github.com/dnsimple/dnsimple-support/pull/1258/commits/9f0f9f294517887d36cfa99483ccf0c87282f998

**Review by weppos — APPROVED** [review]

**Pull request merged**